site/docs/maintenance.md

```diff
@@ -130,3 +130,25 @@ table.rewriteManifests()
 
 See the [`RewriteManifestsAction` Javadoc](/javadoc/master/org/apache/iceberg/actions/RewriteManifestsAction.html) to see more configuration options.
 
+## Appendix
```

Comment:
Do we need this here? It seems like we should link to the API sections that cover this instead of duplicating the content.

Reply:
I feel there's a slight gap between content in `Create a table` in [java-api-quickstart](https://iceberg.apache.org/java-api-quickstart/#create-a-table) page and here, but it would be OK to just link the section if we think end users can infer they need to call `loadTable` instead. WDYT?

Reply:
I agree about the API quickstart. Now that Spark SQL supports DDL operations, we should start in that section with loading a table. At the top, we can point to the "Getting started" page with SQL create table and the section on creating tables, but after that talk about how to load tables with the API.

And linking to the quickstart is good for this, since we want to point to the right docs, not copy information. Thanks!